monai/transforms/spatial/dictionary.py

```diff
@@ -503,16 +530,29 @@ def __init__(
         """
         Args:
             keys: keys of the corresponding items to be transformed.
-            spatial_size (3 ints): specifying output image spatial size [h, w, d].
+            sigma_range: a Gaussian kernel with standard deviation sampled from
+                ``uniform[sigma_range[0], sigma_range[1])`` will be used to smooth the random offset grid.
+            magnitude_range: the random offsets on the grid will be generated from
+                ``uniform[magnitude[0], magnitude[1])``.
+            spatial_size: specifying output image spatial size [h, w, d].
```

Comment:
If the components of the `spatial_size` are non-positive values, the transform will use the corresponding components of `img` size. For example, `spatial_size=(32, -1)` will be adapted to `(32, 64)` if the second spatial dimension size of `img` is `64`.